inspirehep/ext/jinja_filters/record.py

```diff
@@ -390,3 +390,99 @@ def citation_phrase(count):
             return 'Cited 1 time'
         else:
             return 'Cited ' + str(count) + ' times'
+
+    @app.template_filter()
+    def conference_info(record):
+        out = ''
+        title_exists = False
+        if 'publication_info' in record:
+            from invenio_ext.es import es
+            for element in record['publication_info']:
+                if 'journal_title' in element:
+                    title_exists = True
+                if 'conference_recid' in element \
+                        and 'parent_recid' in element:
+                    conference_rec = es.get(
```

Comment:
Can you use [get_source](https://elasticsearch-py.readthedocs.org/en/master/api.html#elasticsearch.Elasticsearch.get_source) instead ? It will avoid having to access the `_source` key and you can specify which fields to get.